application/templates/cms/create_chart_2.html

```diff
@@ -392,17 +402,20 @@ <h2 class="heading-medium">Error</h2>
                     </div>
                 </div>
                 <div id="preview_container">
-                    <h2 class="heading-medium">Preview</h2>
+                    <h2 class="heading-medium builder-section-number">Chart title</h2>
                     <div class="chart-builder-section">
-                        <textarea id="chart_title" class="heading-small interactive-text-input" rows="2" data-module="autoresize no-newlines"
-                               placeholder="[Click to add chart title]"
+                      <span class="form-hint">For example, ‘Percentage of households who own their home by ethnicity and socio-economic group’</span>
+                      {% include 'forms/extended_hints/_chart_and_table_title.html' %}
+                        <textarea id="chart_title" class="full-width" rows="2" data-module="autoresize no-newlines"
```

Comment:
Are we ever likely to replace these work WTForms do you think?

Reply:
One day, hopefully.